src/ensembl/src/shared/checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+
+import defaultStyles from './Checkbox.scss';
+
+export enum CheckboxStatus {
+  CHECKED = 'checked',
+  DISABLED = 'disabled',
+  UNCHECKED = 'unchecked'
+}
+
+type Props = {
+  onClick?: () => void;
+  status: CheckboxStatus;
```

Comment:
While I understand the appeal of an enum, I think here it would be better to use two props: `checked: boolean` and `disabled?: boolean`, again to make the behaviour of the component closer in line with behaviour of the native input type checkbox.

Reply:
That's done.